openlibrary/catalog/amazon/amazon_to_arc.py

```diff
@@ -24,9 +24,9 @@ def get(sock, host, url):
         if chunk_size == 0:
             break
         print(len(fp.read(chunk_size)))
-        print(repr(fp.read(2))
```

Comment:
thank you :) this was flat out broken

Reply:
I'm not sure /catalog/amazon is used by the main site in any way. There seems to be multiple issues with that code.